openedx/core/djangoapps/credit/models.py

```diff
@@ -201,6 +201,19 @@ class CreditRequirementStatus(TimeStampedModel):
     class Meta(object):  # pylint: disable=missing-docstring
         get_latest_by = "created"
 
+    @classmethod
+    def get_statuses(cls, requirement, username):
+        """ Get credit requirement statuses of given requirement and username
+
+        Args:
+            requirement(CreditRequirement): The identifier for a requirement
+            username(str): username of the user
+
+        Returns:
+            CreditRequirementStatus object
+        """
+        return cls.objects.filter(requirement_id__in=requirement, username=username)
```

Comment:
Update the docstring and arguments to reflect the fact that requirements should be a list/queryset of `CreditRequirement` objects.